src/SeaIceThermodynamics/HeatBoundaryConditions/bottom_heat_boundary_conditions.jl

```diff
@@ -30,11 +30,11 @@ is related to the sum of `fluxes` into the ice-water interface,
 """
 IceWaterThermalEquilibrium(; salinity=0) = IceWaterThermalEquilibrium(salinity)
 
-@inline bottom_temperature(i, j, grid, bc::PrescribedTemperature, args...) = @inbounds bc.temperature[i, j]
-@inline bottom_temperature(i, j, grid, bc::PrescribedTemperature{<:Number}, args...) = bc.temperature
+@inline bottom_temperature(i, j, k, grid, bc::PrescribedTemperature, args...) = @inbounds bc.temperature[i, j]
+@inline bottom_temperature(i, j, k, grid, bc::PrescribedTemperature{<:Number}, args...) = bc.temperature
```

Comment:
does this mean that "bottom" can imply different `k`?

Reply:
what does this change have to do with the purpose of the PR?

Reply:
that is a typo leftover from changing many functions from `i, j` to `i, j, k`. I have changed it back